src/Cli/Microsoft.TemplateEngine.Cli/Commands/SymbolStrings.resx

```diff
@@ -126,6 +126,59 @@
   <data name="Command_Alias_Description" xml:space="preserve">
     <value>Creates or displays defined aliases.</value>
   </data>
+  <data name="Command_Details_Description" xml:space="preserve">
```

Comment:
This file is intended only for command symbols localizations. All other strings should be moved to `LocalizableStrings`.